.travis.yml

```diff
@@ -5,6 +5,7 @@ node_js:
   - "4"
   - "6"
   - "7"
+  - "8"
```

Comment:
yes, we should not immediately show warnings to folks on node@7 (in a patch release)

We should remove node@7 on master after the next beta branch point